Dockerfile

```diff
@@ -67,7 +67,7 @@ LABEL org.opencontainers.image.title="SUSE Application Collection" \
     org.opencontainers.image.vendor="SUSE LLC" \
     com.docker.desktop.extension.api.version="0.3.4" \
     com.docker.extension.categories="kubernetes,utility-tools" \
-    com.docker.desktop.extension.icon="https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/logo.svg" \
+    com.docker.desktop.extension.icon="https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/rancher-logo-cow-blue.png" \
```

Comment:
What if we use the `.svg` version instead? Right now it is in the root of the repository. We could move it to `assets/`.

If you do so, we would need to update the line 108 of this `Dockerfile`.